pkg/scsi/lun.go

```diff
@@ -43,8 +43,8 @@ func NewSCSILu(device_uuid uint64, path string, online bool) (*api.SCSILu, error
 		PerformCommand: luPerformCommand,
 		DeviceProtocol: sbc,
 		Storage:        backing,
-		BlockShift:     api.DefaultBlockShift,
-		UUID:           device_uuid,
+		BlockShift:     bs.BlockShift,
```

Comment:
Shall we set `BlockShift` as `api.DefaultBlockShift` if the user does not set `BlockShift`?

Reply:
A fallback to the default value happens in sbc.go:96. Do you want to do it here as well?